compiler+runtime/include/cpp/jank/runtime/obj/inst.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <chrono>
+
+#include <jank/runtime/object.hpp>
+
+namespace jank::runtime::obj
+{
+  using inst_ref = oref<struct inst>;
+  using inst_time_point = std::chrono::time_point<std::chrono::system_clock>;
+
+  struct inst : gc
+  {
+    static constexpr object_type obj_type{ object_type::inst };
+    static constexpr bool pointer_free{ false };
+
+    inst();
+    inst(jtl::immutable_string const &s);
+
+    /* behavior::object_like */
+    bool equal(object const &) const;
+    jtl::immutable_string to_string() const;
+    void to_string(jtl::string_builder &buff) const;
+    jtl::immutable_string to_code_string() const;
+    uhash to_hash() const;
+
+    object base{ obj_type };
+    jtl::ref<inst_time_point> value;
```

Comment:
No need for the `ref` here. This was used with the uuid in order for the forward declaration to happen (if you hold a pointer to some `T`, you don't need to know the size of the `T`, since all pointers are the same size, so `T` can be forward declared). Since we're not forward declaring, just hold the actual `inst_time_point` as a value.